.github/workflows/CI-orchestrator.yml

```diff
@@ -48,6 +52,7 @@ jobs:
         check-lockfile-job,
         build-job,
         test-job,
+        test-bench-job,
```

Comment:
This job reference needs to be added to the block of 91-103 lines as well since it is part of overall status now